rs/universal_canister/impl/BUILD.bazel

```diff
@@ -43,6 +43,42 @@ rust_canister(
     deps = LIB_DEPENDENCIES + DEPENDENCIES,
 )
 
+genrule(
+    name = "universal_canister_serialized_module",
+    testonly = False,
+    srcs = [":universal_canister"],
+    outs = ["universal_canister.module"],
+    cmd_bash = " && ".join([
+        "{pigz} --decompress --no-name {input_wasm} --stdout > $@.unzip",
+        "{instrument_wasm} $@.unzip -a serialized-module > $@",
+    ])
+        .format(
+        input_wasm = "$(location :universal_canister)",
+        instrument_wasm = "$(location //rs/embedders:instrument-wasm)",
+        pigz = "$(location @pigz)",
+    ),
+    message = "Compiling universal canister with Wasmtime",
+    tools = [
+        "//rs/embedders:instrument-wasm",
+        "@pigz",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+# genrule(
```

Comment:
A leftover?

Reply:
Sorry, yeah. Cleaned up some things in the last commit.